landoapi/revisions.py

```diff
@@ -25,7 +25,7 @@
 logger = logging.getLogger(__name__)
 
 
-def gather_involved_phids(revision: dict) -> set[str]:
+def gather_involved_phids(revision: dict, diffs: dict[str, dict]) -> set[str]:
     """Return the set of Phobject phids involved in a revision.
```

Comment:
I think this docstring needs to be updated to reflect the new `diff` parameter. The current one was scoped for a particular "revision" dict.